src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -32,6 +32,8 @@
   "AnalyseVideoFiles": "Analyse video files",
   "Analytics": "Analytics",
   "AnalyticsEnabledHelpText": "Send anonymous usage and error information to Radarr's servers. This includes information on your browser, which Radarr WebUI pages you use, error reporting as well as OS and runtime version. We will use this information to prioritize features and bug fixes.",
+  "AndNot": "and not",
+  "Announced": "Announced",
```

Comment:
Where are you using this? If not, get rid of it.

Reply:
If I recall properly, I added it due to to review comments: https://github.com/Radarr/Radarr/pull/4924#discussion_r486584615

If I recall correctly, it is and was being used - I never actually used it, I just renamed the file where it was being used and there just wasn't a translation for it before. As per the austinwbest review comments - I added it to the en.json file.

Reply:
Yep you are right, missed that use